src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5427,6 +5427,21 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_ImplicitCopyInReadOnlyMember_Title" xml:space="preserve">
     <value>Call to non-readonly member from a 'readonly' member results in an implicit copy.</value>
   </data>
+  <data name="ERR_StaticMemberCantBeReadOnly" xml:space="preserve">
+    <value>Static member '{0}' cannot be 'readonly'.</value>
+  </data>
+  <data name="ERR_AutoPropertyCantBeReadOnly" xml:space="preserve">
+    <value>Auto-implemented property or accessor '{0}' cannot be marked 'readonly'.</value>
+  </data>
+  <data name="ERR_InvalidPropertyReadOnlyMods" xml:space="preserve">
+    <value>Cannot specify 'readonly' modifiers on both property or indexer '{0}' and its accessors.</value>
```

Comment:
This sentence doesn't parse right to me. I think it should say "Duplicate 'readonly' modifier on both the property and an accessor. Remove the modifier on the property or the modifier on the accessor."